Client/mods/deathmatch/logic/rpc/CColShapeRPCs.cpp

```diff
@@ -55,17 +55,24 @@ void CColShapeRPCs::UpdateColPolygonPoint(CClientEntity* pSource, NetBitStreamIn
 void CColShapeRPCs::AddColPolygonPoint(CClientEntity* pSource, NetBitStreamInterface& bitStream)
 {
     SPosition2DSync size(false);
-    int             iPointIndex;
-    if (bitStream.Read(&size) && bitStream.Read(iPointIndex))
```

Comment:
I suspect polygon point update function calls will now fail against older clients, but I think it's mostly fine because this is a beta feature anyway.